File: tests/fixtures/lang_samples/python/nested.py
def deeply_nested(x):
    if x > 0:
        for i in range(x):
            if i % 2 == 0:
                while i > 0:
                    try:
                        if i == 5:
                            return i
                    except Exception:
                        pass
                    i -= 1
    return 0


def shallow(x):
    return x + 1

File: tests/fixtures/lang_samples/go/nested.go
package sample

func DeeplyNested(x int) int {
	if x > 0 {
		for i := 0; i < x; i++ {
			if i%2 == 0 {
				for i > 0 {
					if i == 5 {
						return i
					}
					i--
				}
			}
		}
	}
	return 0
}

func Shallow(x int) int {
	return x + 1
}
